ARScript.R

```diff
@@ -0,0 +1,6 @@
+library(dplyr)
+
+
+table(iris$Sepal.Length)
```

Comment:
I would suggest adding some better comments to explain this function - what is table(), how does it work????